lib/rubocop/config_loader_resolver.rb

```diff
@@ -12,7 +12,12 @@ def resolve_requires(path, hash)
         if r.start_with?('.')
           require(File.join(config_dir, r))
         else
-          require(r)
+          begin
+            require(r)
+          rescue LoadError
+            puts "Unable to load '#{r}', are you missing a gem or a file?"
+            exit
```

Comment:
[`abort`](http://ruby-doc.org/core-2.5.0/Kernel.html#method-i-abort) seems appropriate here since it writes to STDERR and terminates the process with a exit code that indicates failure.